llvm/test/ExecutionEngine/OrcLazy/debug-objects.ll

```diff
@@ -0,0 +1,62 @@
+; REQUIRES: native && x86_64-linux
+
+; In-memory debug-objects contain DWARF
+;
+; RUN: lli --jit-linker=rtdyld  --orc-lazy-debug=jit-debug-objects %s | llvm-dwarfdump --diff - | FileCheck %s
+; RUN: lli --jit-linker=jitlink --orc-lazy-debug=jit-debug-objects %s | llvm-dwarfdump --diff - | FileCheck %s
+;
+; CHECK: -:	file format elf64-x86-64
+; TODO: Synthesized Mach-O objects error out with:
+;       truncated or malformed object (offset field of section 8 in
+;       LC_SEGMENT_64 command 0 extends past the end of the file)
```

Comment:
@lhames I guess this could be fixed in the Mach-O debugger support plugin? Then we can re-enable this test as well.